specification/azure-kusto/resource-manager/readme.typescript.md

```diff
@@ -7,6 +7,6 @@ Please also specify `--typescript-sdks-folder=<path to root folder of your azure
 typescript:
   azure-arm: true
   package-name: "@azure/arm-kusto"
-  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-kusto"
```

Comment:
I like using kusto here but I don't like that the folder name is inconsistent in the specs repo :(